external-crates/move/move-compiler/tests/move_check/typing/unused_struct_field.move

```diff
@@ -0,0 +1,47 @@
+module 0x42::unused_fields {
```

Comment:
Can we add a test where the struct is unused? Then I think _all_ of the fields should be reported unused?